.github/workflows/ci.yml

```diff
@@ -9,6 +9,14 @@ on:
   pull_request:
```

Comment:
I think it is called `ci.yaml` in all of the flutter repositories? Not that it matters much, but that is a way more common name in GitHub actions in general too.

Reply:
I don’t really care about the name, we got some back and forth on CI vs QA in the past. As long as we align all our repo (most of Ubuntu repos have qa.yaml)

Reply:
In the Ubuntu repos there are 22 called *ci.yaml and 9 called *qa.yaml, and in the Canonical repos there are 
166 called *ci.yaml and 0 called *qa.yaml, so I think that it would be easier to align by naming the `qa` ones `ci` instead? That would also align more with the common standard on GitHub.

https://github.com/search?q=org%3Aubuntu+path%3A**%2F*ci.y*ml&type=code
https://github.com/search?q=org%3Aubuntu+path%3A**%2F*qa.y*ml&type=code
https://github.com/search?q=org%3Acanonical+path%3A**%2F*ci.y*ml&type=code
https://github.com/search?q=org%3Acanonical+path%3A**%2F*qa.y*ml&type=code

Reply:
I’m fine with it (just a note: in reality, the actual number differs due to non public repos, but that doesn’t change much the numbers).

The only thing to take into account, is that some repos have multiple workflows being part of "CI" technically, but needed to be split for clarity, so not everything can be renamed that easily, but as long as we standardize (I think those could have some domains like `ci-<domain>` for instance. Let’s go for `ci.yaml` then and calling those the ones running on PR only!

Reply:
The real numbers on public ubuntu repos are: 8 `ci.y*ml` in github ubuntu org vs 9 `qa.y*ml`. But anyway, as told, let’s go with `ci.yaml`.

Reply:
Maybe that is a pretty good idea, to split it up to `ci-flutter.yaml` and `ci-golang.yaml`?